app/src/org/commcare/connect/ConnectJobHelper.kt

```diff
@@ -85,23 +84,31 @@ object ConnectJobHelper {
         object : ConnectApiHandler<DeliveryAppProgressResponseModel>() {
             override fun onSuccess(deliveryAppProgressResponseModel: DeliveryAppProgressResponseModel) {
                 if (deliveryAppProgressResponseModel.updatedJob) {
+                    FirebaseAnalyticsUtil.reportCccApiDeliveryProgress(true,"Started a delivery job")
```

Comment:
All event parameters should be underscored constant codes rather than a free text  (start_delivery, end_delivery etc)